app/blueprints/fund/forms.py

```diff
@@ -37,16 +38,62 @@ def __init__(self, name: str, _prefix, _translations, label: str, _form, source_
 
 class FundForm(FlaskForm):
     fund_id = HiddenField("Fund ID")
-    name_en = StringField("Grant name", validators=[DataRequired()])
-    name_cy = StringField("Grant name (Welsh)")
-    title_en = StringField("Application heading", validators=[DataRequired()])
-    title_cy = StringField("Application heading (Welsh)")
+    welsh_available = RadioField(
+        "Is this grant available in Welsh?",
+        choices=[("true", "Yes"), ("false", "No")],
```

Comment:
When we do boolean radio fields, I think it's nice to add a `coerce` step that gives us true booleans:

```suggestion
        choices=[("true", "Yes"), ("false", "No")],
        coerce=lambda value: value == 'true',
```

Then in the view/route handler we can just check `form.welsh_available.data` for truthiness, rather than checking for `== "true"`